jenkins.yaml

```diff
@@ -50,9 +50,9 @@ tool:
     installations:
       - name: "maven"
         properties:
-          - installSourceProperty:
+          - installSource:
               installers:
-              - mavenInstaller:
+              - maven:
                   id: 3.6.0
```

Comment:
3.6.3?

Reply:
probably, but I don't want to restart the installation just to upgrade maven for now

Reply:
I don't get the link between putting the last version in this file and restarting a server. Can't we check locally for example?

Reply:
I don't see how. Testing this fully requires the jenkins server + 2 slaves, with one of them having the good btrfs configuration...
This PR reflects the last deployment, which has been tested. I propose to open an other PR to upgrade maven and maybe sbt

Reply:
https://github.com/linagora/james-jenkins2/pull/11